index.bs

```diff
@@ -889,6 +792,102 @@ user consent to a specific transaction. The structure of these signatures is def
     [[#op-get-assertion]].
 </div>
 
+## Additional options for Attestation (dictionary <dfn dictionary>MakeCredentialOptions</dfn>) ## {#dictionary-makecredentialoptions}
```

Comment:
yeah, as i noted in review of PR #398 maybe these are not "options" but are parameters, some of which are optional ?